libV2/warp10.js

```diff
@@ -15,11 +21,36 @@ class Warp10Client {
         this._readToken = (config && config.readToken) || 'readTokenCI';
         this._nodeId = (config && config.nodeId) || _config.nodeId;
         const proto = (config && config.tls) ? 'https' : 'http';
-        const host = (config && config.host) || 'localhost';
-        const port = (config && config.port) || 4802;
         const requestTimeout = (config && config.requestTimeout) || 10000;
         const connectTimeout = (config && config.connectTimeout) || 10000;
-        this._warp10 = new Warp10(`${proto}://${host}:${port}`, requestTimeout, connectTimeout);
+        if (config && config.hosts) {
+            this._clients = config.hosts.map(h => {
```

Comment:
I think this arrow function could be written as `({ host, port }) => new Warp10(...)`